khelo/lib/ui/flow/matches/match_detail/components/match_detail_scorecard_view.dart

```diff
@@ -78,20 +87,12 @@ class MatchDetailScorecardView extends ConsumerWidget {
                   ?.squad
                   .toList();
 
-              List<List<int>> powerPlays = [
-                if (state.match?.power_play_overs1.isNotEmpty ?? false)
-                  state.match!.power_play_overs1,
-                if (state.match?.power_play_overs2.isNotEmpty ?? false)
-                  state.match!.power_play_overs2,
-                if (state.match?.power_play_overs3.isNotEmpty ?? false)
-                  state.match!.power_play_overs3,
-              ];
-
               return _teamTitleView(context,
                   teamName: _getTeamNameByTeamId(state, overs?.team_id ?? ""),
                   over: overs ?? const OverSummary(),
-                  initiallyExpanded: state.expandedTeamScorecard
-                      .contains(overs?.team_id ?? ""),
+                  initiallyExpanded: groupOversByInnings.length == 1 ||
```

Comment:
- if I close the card and change tab then it should remain the closed but it 's opened, which is not correct behaviour.
- also the icon is not animating for open and closed states, can you fix it as well.
![iconAnimationIssue](https://github.com/canopas/khelo/assets/122426509/a987cd9f-9d1b-43dd-a6ed-c62e403a32b2)